src/assets/stylesheets/variables/_layout.scss

```diff
@@ -0,0 +1,6 @@
+$grid-columns: 12;
+$grid-gutter-width: 30px;
```

Comment:
Both of these values are boostrap default values. I guess it's save to remove these grid variables.